intervention_compassion/__manifest__.py

```diff
@@ -30,7 +30,7 @@
 # pylint: disable=C8101
 {
     'name': 'Compassion Interventions',
-    'version': '10.0.1.1.0',
+    'version': '10.0.1.2.0',
```

Comment:
On my db the migration doesn't work because the intervention inside doesn't exist anymore in US db.